### PR TITLE
add intro pages

### LINKED ISSUE #1616: Implement docs introduction

### LINKED ISSUE #1617: Implement docs authorization page

### LINKED ISSUE #1618: Implement docs pagination page

### LINKED ISSUE #1619: Implement docs Sorting and Filtering page

### LINKED ISSUE #1620: Implement docs errors page

### PR DESCRIPTION
**Pull request by abemassry**
closes #1616 
closes #1617
closes #1618
closes #1619 
closes #1620

### CODE REVIEW

File: docs/src/layouts/Authentication.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+  return (
+    <div>
+      <div className="row reference white">
+        <div className="col-lg-6 text-justify left">
+          <h4>Authentication
+            <span className="anchor" id="authentication">&nbsp;</span>
+          </h4>
+          <p>
+          To use some API endpoints authentication is required. This is indicated by the
+          <span className="text-muted"><i className="fa fa-lock"></i> Authenticated</span> notation
```
Comment (eatonphil) [inline docs/src/layouts/Authentication.js]: Should probably drop this for now?

File: docs/src/layouts/Authentication.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+const LOGIN_ROOT = 'login.alpha.linode.com';
+export default function Authentication() {
+  return (
+    <div>
+      <div className="row reference white">
+        <div className="col-lg-6 text-justify left">
```
Comment (eatonphil) [inline docs/src/layouts/Authentication.js]: Drop this entire div everywhere please and the one above it. We have no need for the column layout.

File: docs/src/components/intros/Filtering.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+const API_ROOT = 'api.alpha.linode.com';
+const API_VERSION = 'v4';
```
Comment (eatonphil) [inline docs/src/components/intros/Filtering.js]: Could we move these into docs/src/constants.js?

**Comment by mparke** [comment]
Follow up: https://github.com/linode/manager/issues/1724

File: docs/src/components/intros/Access.js [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        <Table
+          className="Table--secondary"
+          columns={[
+            { dataKey: 'parameter', label: 'Parameter', headerClassName: 'ParameterColumn' },
+            { dataKey: 'description', label: 'Description', headerClassName: 'DescriptionColumn' }
+          ]}
+          data={[
+            {
+              parameter: 'client_id',
+              description: 'Required. Your appliication\'s client ID.'
```
Comment (eatonphil) [inline docs/src/components/intros/Access.js]: apliication

File: docs/src/components/intros/Introduction.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        <dl>
+          <dt>GET</dt>
+          <dd>Gets information about a resource or resources</dd>
+          <dt>PUT</dt>
+          <dd>Edits information about a resource</dd>
+          <dt>POST</dt>
+          <dd>Creates a new resource</dd>
+          <dt>DELETE</dt>
+          <dd>Deletes a resource</dd>
+        </dl>
```
Comment (eatonphil) [inline docs/src/components/intros/Introduction.js]: Could we replace this with a table like the status / codes in the Errors section?

File: docs/src/components/intros/Errors.js [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+          probably going to be running around with our hair on fire if you ever
+          see them.
+        </p>
+        <p>
+          Every request that returns errors will look something like this:
+        </p>
+        <pre>
+          <code>
+{`{
+"errors": [
```
Comment (eatonphil) [inline docs/src/components/intros/Errors.js]: +2 indentation?

File: docs/src/layouts/Layout.js [hunk 7]
```diff
@@ -18,9 +18,10 @@ export default function Layout(props) {
               <h3>Getting Started</h3>
               <ul>
                 <li><Link to="/introduction">Introduction</Link></li>
-                <li><Link to="/authorization">Authorization</Link></li>
+                <li><Link to="/access">Access</Link></li>
                 <li><Link to="/pagination">Pagination</Link></li>
                 <li><Link to="/filtering">Filtering</Link></li>
```
Comment (eatonphil) [inline docs/src/layouts/Layout.js]: Filtering & Sorting?

File: docs/webpack.config.dev.js [hunk 8]
```diff
@@ -28,6 +28,9 @@ module.exports = {
       'process.env': {
         'NODE_ENV': JSON.stringify(process.env.NODE_ENV)
       },
+      'ENV_API_ROOT': JSON.stringify(process.env.API_ROOT),
+      'ENV_LOGIN_ROOT': JSON.stringify(process.env.LOGIN_ROOT),
+      'ENV_API_VERSION': JSON.stringify(process.env.API_VERSION),
```
Comment (eatonphil) [inline docs/webpack.config.dev.js]: Not much sense having this in here because it's not really configurable in the same way...

Comment (eatonphil) [inline docs/webpack.config.dev.js]: Eh, whatever. Either way works.

**Comment by eatonphil** [comment]
Just noticed we really need borders between columns. Hard to read this much text without them.

**Comment by mparke** [comment]
See https://github.com/abemassry/manager/pull/19 for addressed PR Feedback

**Pull request merged**